ament_flake8/ament_flake8/main.py

```diff
@@ -139,8 +139,10 @@ def generate_flake8_report(config_file, paths, excludes, max_line_length=None):
         return generate_flake8_report(config_file, paths, excludes, max_line_length)
 
     flake8_argv = []
-    flake8_argv.append('--config={0}'.format(config_file))
-    flake8_argv.append('--exclude={0}'.format(excludes))
+    if config_file is not None:
+        flake8_argv.append('--config={0}'.format(config_file))
+    if excludes is not None:
```

Comment:
Changing the check to `if not excludes:` will give consistent behaviour with the other linters in the event the list is empty.  Unless there's a reason this check is more appropriate, could you update it?